cmd/bundle.rb

```diff
@@ -77,6 +77,11 @@ def bundle_args
       switch "--describe",
              description: "`dump` adds a description comment above each line, unless the " \
                           "dependency does not have a description."
+      switch "--cask-force",
```

Comment:
I wonder if this would be better as a more general option of `--force` that applies to both formulae and casks?

Reply:
@jacobbednarz Agreed and I've moved it from being dump-time to install-time.